### ISSUE TITLE
Helm popup size doesn't change untill i reload my doom config

### ISSUE THREAD

**Issue by Atreus-Tantalus** [hlissner/doom-emacs#1304]
Command: `(set-popup-rule! "^\\*helm" :size 0.5)`
I tried using `(after!)` for both helm and popup modules, but it takes effect only after i reload doom config via `(doom/reload)` command.

**Comment by hlissner** [comment]
```lisp
(after! helm
  (set-popup-rule! "^\\*helm" :size 0.5))
```

Is this what you tried?

**Comment by Atreus-Tantalus** [comment]
yes, i tried using after! macro for both helm and popup module. Still helm popup size doesn't change until i refresh doom

**Comment by hlissner** [comment]
Are you still experiencing this issue? I am unable to reproduce this.

**Comment by brotzeit** [comment]
I can reproduce it. When I put 

```elisp
(after! helm
  (set-popup-rule! "^\\*helm" :size 0.5))
```
in my config, it doesn't change the initial size. I have to evaluate it manually or run `doom/reload`.

**Comment by TimQuelch** [comment]
I am also seeing this behaviour with the same configuration.

Doesn't work on first startup, but if I manually eval the snippet it does work.

Something to do with the initialisation order?

**Comment by shkodm** [comment]
Just had the same problem. In my case, the issue was that there is a separate popup rule set for `helm-rg` which was set in doom's own original configuration for helm in a separate `(after! helm-rg`). 

I fixed by having my helm popup config as: 
```
(after! (:and helm helm-rg)
  (set-popup-rule! "^\\*helm" :size 0.5))
```
